internal/throttler/remote/throttler.go

```diff
@@ -34,12 +34,11 @@ const (
 )
 
 var (
-	errorUUIDNotSet = errors.New("Throttler uuid must be set")
+	errorClientIDNotSet = errors.New("Throttler ClientID must be set")
 )
 
 type creditResponse struct {
-	Operation string  `json:"operation"`
-	Credits   float64 `json:"credits"`
+	BalancesByOperation map[string]float64 `json:"balancesByOperation"`
```

Comment:
I still have the same 2 comments as before, I'd prefer this to not be a map but an array so it's more extensible in the future. The internal representation inside the client can be a map but json should be array. How much of a headache is this to change?

Reply:
I wasn't sure if it was using maps at all or just a very unstructured map with no fields to encapsulate it. I don't mind a list so much just don't think it's inherently more extensible than a map. Some protocols implements maps while others don't. Thrift seems to prefer lists.

Reply:
in the future, if we use the creditResponse to handle both debug throttling and normal sampling throttling, a 1 to 1 mapping won't work. You can either change it to a json list or map[string]struct{}. Where does thrift come into this? Looks like we don't have a thrift definition for the credits payload.